tokens/config.js

```diff
@@ -1,6 +1,60 @@
 module.exports = {
   source: ["tokens/*.json"],
+  parsers: [{
+    pattern: /\.json$/,
+    parse: ({ filePath, contents, ...rest }) => {
+      contents = JSON.parse(contents);
+      const categories = Object.entries(contents);
+      for (const [category, categoryValue] of categories) {
+        const types = Object.entries(categoryValue);
+
+        for (const [type, typeValue] of types) {
+          const items = Object.entries(typeValue);
+
+          for (const [item, itemValue] of items) {
+            if (["gradient", "elevation"].includes(type) && itemValue && !itemValue.type) {
+              const subitems = Object.values(itemValue);
+              contents[category][type][item] = {
+                ...subitems[0],
+                extensions: itemValue.extensions
+              };
+
+              contents[category][type][item].value = subitems.filter(v => v && v.value).map(v => v.value);
+            }
+          }
+        }
+      }
+      return contents;
```

Comment:
Perhaps a little comment about what this parser does, since it doesn't have a function or file name?

Reply:
Good point. In addition, should this also be pulled into its own file? I debated on that.